components/app-launcher/docs.json

```diff
@@ -11,5 +11,5 @@
             "component": "tile"
         }
     ],
-    "url-slug": "app-launchers"
+    "url-slug": "app-launcher"
```

Comment:
The doc site uses the singular version for some reason, modified it to reflect this.